### ISSUE TITLE
review info messages to users

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#307]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/307) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

review all Info messages.

e.g,

```
#!text

INFO: Default MSS config file missing.
```

it means

```
#!text

INFO: Default MSS configuration in place, no user settings.
```

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

message to user improved, fixes #307

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #541)

message to user improved, fixes #307
